src/app/models/indicator.model.ts

```diff
@@ -10,6 +10,7 @@ export class Indicator {
     available_units?: string[];
     default_units?: string;
     parameters?: IndicatorParameter[];
+    thresholdIndicator: boolean;
```

Comment:
Went with a utility module, since it's just checking string membership in arrays.

Although it is a derived property, for performance reasons, it would be more efficient to cache the results somewhere, and the indicator object would seem to be an obvious place for that. Efficiency might not be an issue, though.